### PR TITLE
CSP-911 Admin Aan CalendarEventSearchResults

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.AdminAan.Domain/Constants/EventStatus.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace SFA.DAS.Admin.Aan.Domain.Constants;
+public static class EventStatus
+{
+    public static string Published => "Published";
```
Comment (DevenKShah) [inline src/SFA.DAS.AdminAan.Domain/Constants/EventStatus.cs:4]: Can this be converted to const?

File: src/SFA.DAS.AdminAan.Domain/OuterApi/Responses/CalendarEventSummary.cs
Comment (DevenKShah) [inline src/SFA.DAS.AdminAan.Domain/OuterApi/Responses/CalendarEventSummary.cs:1]: Do we need a domain project? Can all these be in the application?

File: src/SFA.DAS.Admin.Aan.Application/OuterApi/Regions/Calendar.cs
Comment (DevenKShah) [inline src/SFA.DAS.Admin.Aan.Application/OuterApi/Regions/Calendar.cs:1]: Why is this in Regions folder?

**Review by DevenKShah — CHANGES_REQUESTED** [review]
@MarkFCain Can we loose the domain project and use nuget package for shared components that are copied as is please, sorry for rework.

File: src/SFA.DAS.Admin.Aan.Application/OuterApi/Regions/Calendar.cs
Comment (MarkFCain) [reply src/SFA.DAS.Admin.Aan.Application/OuterApi/Regions/Calendar.cs:1]: error, removed

File: src/SFA.DAS.AdminAan.Domain/OuterApi/Responses/CalendarEventSummary.cs
Comment (MarkFCain) [reply src/SFA.DAS.AdminAan.Domain/OuterApi/Responses/CalendarEventSummary.cs:1]: moved to application

File: src/SFA.DAS.AdminAan.Domain/Constants/EventStatus.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.AdminAan.Domain/Constants/EventStatus.cs:4]: yes

**Review by DevenKShah — COMMENTED** [review]
Domain project should be removed and CSS file to be restored to original format.

**Review by DevenKShah — DISMISSED** [review]

**Review by YuvrajPatil-DFE — APPROVED** [review]

**Pull request merged**